### DISCUSSION TITLE
Low power mode feature

### DISCUSSION THREAD

**Discussion by masterzone3d** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1216] [General]
Hi to all, I would like to know more about "Low power mode" toggle into the Al dente settings.
What kind of relationship is there from this Aldente option and the MacOS system low power mode set on "automatic"?...

example: in this moment I'm on battery and the "low power mode" is disable by default into the al dente settings, se below:

<img width="420" alt="Screenshot 2024-04-21 at 11 44 11" src="https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/assets/132667740/4fd1ff5d-8d15-4c1d-a566-40f9ef555a42">

does it mean that I'm on wrong settings right now? why does this feature become "manual" and not "automatic" by the system?
It sincerely drive me confused :)

**Comment by MatthiasKerbl** [comment]
Hi @masterzone3d,

Depending on your own preference and needs you can decide if you want to manually control the macOS Low Power Mode through AlDente or automatically through macOS.
Please keep in mind to completely deactivate the Low Power Mode in whichever option you choose not to use.

Best,
Matthias

**Reply by masterzone3d** [reply]
thanks for your replay...

mmm ok but it not clear the option...right now I don't touched the MacOs options, so now all is set to automatic:

![automatic](https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/assets/132667740/b2648b4e-dd6b-4e91-870c-0656b1a22c50)

Usually the low power mode features starts automatically when you unplug the magsafe from the MacBook...why would I need to manually turn on and off via AlDente? Why I need to "simulate" the unplug when I'm plugged in? During the "battery low power mode" you got a screen dimming and quietly fan and also (maybe) a CPU or GPU throttle...

Sorry for stupid questions but I'm trying to understand deeply this feature...

**Reply by MatthiasKerbl** [reply]
Hi @masterzone3d,

Some people prefer to manually switch to Low Power Mode depending on their usage. However,  if you prefer to let macOS take care of it, that's also fine.

Best,
Matthias

**Reply by masterzone3d** [reply]
ah ok...sounds good...I was curious about why I need to switch to low power mode if I'm plugged in...
Anyway thanks for your time and greetings for the app...a great peace of software!

**Reply by dvcrn** [reply]
Hi @MatthiasKerbl, I wasn't sure where the right place for a feature request is, but the low power toggle in aldente is great, I love the right-click action. Can we have a indicator in the menu bar if low power mode is on and off? The only way to see it is if additional stats are enabled, then it has a "Low Power Mode: Enabled" text entry, but that requires a click on the icon

(And even better, a notification when it switched?)
